spec/support/serializers/allocation_uplift_serializer.rb

```diff
@@ -0,0 +1,9 @@
+class AllocationUpliftSerializer < JSONAPI::Serializable::Resource
+  type "allocation_uplifts"
+
+  belongs_to :allocation
+
+  attributes(*FactoryBot.attributes_for("allocation_uplift").keys)
+
+  attribute :uplifts
```

Comment:
Are both of these lines necessary?

Reply:
No, in fact, this file isn't necessary so I'll remove it now